src/test/java/RequestHandlerTest.java

```diff
@@ -0,0 +1,25 @@
+import com.td.HTTPServer.*;
+
+class MockVerifier implements HTTPVerifierInterface {
```

Comment:
Yep! You could make this mock even simpler by making two, a `MockValidVerifier` and `MockInvalidVerifier` that just return `true` and `false` respectively and inject those into `processRequest`